fixtures/pages-functions-wasm-app/package.json

```diff
@@ -14,7 +14,8 @@
 	},
 	"devDependencies": {
 		"undici": "^5.9.1",
-		"wrangler": "*"
+		"wrangler": "*",
+		"@cloudflare/workers-tsconfig": "workspace:*"
```

Comment:
This uses `workspace:*` but the file above (external-durable-objects-app/package.json` does not.